server/controller/UserController.js

```diff
@@ -651,6 +653,27 @@ router.get("/gliders/get", authToken, async (req, res, next) => {
   }
 });
 
+// @desc Gets the admin panel notifications
+// @route GET /users/adminNotifications
+
+router.get("/adminNotifications", authToken, async (req, res, next) => {
+  try {
```

Comment:
Warum 2x catch?
Das innere catch kannst du dir sparen. Du machst ja in beiden Fällen die gleiche Fehlerbehandlung.

Die Idee des catch in den Endpoints ist es alle Fehler abzufangen und dann den globalen ErrorHandler über das next(error) zu übergeben.

Reply:
das war ein copy paste Fehler